src/assets/styles/components/task-details.scss

```diff
@@ -1,5 +1,185 @@
-.task-details-container {
-  margin-block: 48px;
-  background-color: var(--surface);
-  width: 765px;
+.task-details {
+  padding: 20px;
+  background-color: var(--ds-background-neutral);
+  height: 100%;
+  max-width: 100%;
+  overflow-y: auto;
+  overflow-x: hidden;
+  color: var(--ds-text);
+
+  .btn {
+    color: var(--ds-text);
+  }
+
+  .task-header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100px;
+
+    .group-container {
+      display: flex;
+      margin-left:65px;
+      width: 100%;
+
+      p {
+        display: flex;
+        align-items: center;
+        gap:5px;
+      }
+    }
+
+    .title {
+      display: flex;
+      align-items: center;
+      height: 36px;
+      gap: 10px;
+      width: 100%;
+
+      .title-text {
+        font-size: 20px;
+        width: 100%;
+      }
+    }
+
+    .title-input {
+      display: flex;
+      font-size: 20px;
+      color: var(--ds-text);
+      background-color: var(--surface);
+      font-weight: 600;
+      width: 100%;
+      max-width: 600px;
+      border-radius: 5px;
+      margin-left: -5px;
+      transition: ease-in-out 0.3s;
+      height: 36px;
+      border: none;
+      outline: none;
+
+
+      &:focus {
+        box-shadow: inset 0 0 0 2px var(--ds-border-focused);
+      }
+    }
+  }
+
+  .task-body {
+    display: grid;
+    grid-template-columns: 3fr 1fr;
+    margin-top: 10px;
+
+    .task-body-content {
+      width: 100%;
+
+      .labels-container {
+        display: flex;
+        gap:10px;
+        margin-left: 20px;
+        
+
+        .labels {
+          display: flex;
+          flex-direction: column;
+          margin-inline: 10px;
+          
+
+          span {
+            font-size: 12px;
+            font-weight: 600;
+            margin-bottom: 5px;
+          }
+  
+            div {
+              display: flex;
+              align-items: center;
+              gap: 10px;
+            }
+      }
+      }
+
+      .description-container {
+        
+        .action-title {
+          display: flex;
+          align-items: 'center';
+          justify-content: 'space-between';
+          gap: '10px';
+        }
+
+        .textarea-description {
+          background-color: var('--surface');
+        }
+        margin-top: 20px;
```

Comment:
if the margin-top:20px is to the class description-container, move before the other classes